Cargo.toml

```diff
@@ -26,16 +26,16 @@ pyo3cls = { path = "pyo3cls", version = "=0.6.0-alpha.4" }
 mashup = "0.1.9"
 num-complex = { version = "0.2.1", optional = true }
 inventory = "0.1.3"
-
+pretty_assertions = "0.5.1"
```

Comment:
Is pretty_assertions used as a regular dependency?

Reply:
nope, removed.